SharedProjects/Babylon2GLTF/GLTFExporter.cs

```diff
@@ -482,7 +482,7 @@ private List<GLTFBufferView> SwitchImagesFromUriToBinary(GLTF gltf)
 
             foreach (GLTFImage gltfImage in gltf.ImagesList)
             {
-                var path = Path.Combine(gltf.OutputFolder, Uri.UnescapeDataString(gltfImage.uri));
+                var path = Path.Combine(gltf.OutputFolder, gltfImage.uri);
```

Comment:
Why are we no longer escaping the symbols in our path?

Reply:
it is already unescaped in PathGetRelativePath..this was a repetition.
the gltfimage uri is set only in L176